autoload/neomake/makers/ft/haskell.vim

```diff
@@ -73,9 +73,12 @@ function! neomake#makers#ft#haskell#hdevtools() abort
     if !exists('s:uses_cabal')
         let s:uses_cabal = 0
         if executable('stack')
-            let rootdir = systemlist('stack --verbosity silent path --project-root')[0]
-            if empty(glob(rootdir . '/*.cabal'))
-                let s:uses_cabal = 1
+            let output = neomake#utils#systemlist(['stack', '--verbosity silent', 'path', '--project-root'])
+            if !empty(output)
+                let rootdir = output[0]
+                if empty(glob(rootdir . '/*.cabal'))
+                    let s:uses_cabal = 1
+                endif
```

Comment:
Yah, I think the meaning got inverted here:

https://github.com/neomake/neomake/commit/7542a1c92fa71f99ea102499e00826c406762793#diff-6d67a70692ff8830b142f7e3631900f4

So, should be !empty instead of empty. That said...I don't think hdevtools needs to told to use Stack anymore (its smart enough to use stack on its own), nor does it need the workaround for setting GHC-PACKAGE-PATH anymore, so we could perhaps remove that whole section of code.